### PR TITLE
PR to main from Landing Page

### PR DESCRIPTION
**Pull request by inikasingh21**
Created Landing Page

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -38,88 +23,107 @@ class MyApp extends StatelessWidget {
...
+                        setState(() {
+                          searchQuery = value; //user input being assigned as recieved 
+                          });
+                          } ,
+                        ),
+                      ),
+                      const SizedBox(width: 10),
+                      ElevatedButton(
+                         onPressed: (){
+                      //     Navigator.push(
```
Comment (erickjtorres) [inline lib/main.dart]: remove unnecessary comments

File: lib/main.dart [hunk 2]
```diff
@@ -38,88 +23,107 @@ class MyApp extends StatelessWidget {
...
+                  Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 20.0),
+                  child: Row(
+                    children: [
+                      Expanded(
+                      child: SearchBar(
+                      hintText: 'Search by name, phone, or email',
+                      onChanged: (value) {
+                        setState(() {
+                          searchQuery = value; //user input being assigned as recieved
```
Comment (erickjtorres) [inline lib/main.dart]: don't need this comment

File: lib/main.dart [hunk 3]
```diff
@@ -38,88 +23,107 @@ class MyApp extends StatelessWidget {
...
+                      //       MaterialPageRoute(//go to other pg, find query
+                      //       builder: (context) => ResultsPage(query: searchQuery),
+                      //     ),
+                      //   );
+                       },
+                      style: ElevatedButton.styleFrom(
+                        backgroundColor: const Color.fromARGB(255, 123, 127, 211), // Change the button color here
+                        minimumSize: const Size(50, 50), // Change the size of the button
+                        padding: const EdgeInsets.all(16), // Adjust padding for size
+                        //shape: RoundedRectangleBorder(
```
Comment (erickjtorres) [inline lib/main.dart]: unnecessary

File: lib/main.dart [hunk 4]
```diff
@@ -38,88 +23,107 @@ class MyApp extends StatelessWidget {
...
-      // changed in this State, which causes it to rerun the build method below
-      // so that the display can reflect the updated values. If we changed
-      // _counter without calling setState(), then the build method would not be
-      // called again, and so nothing would appear to happen.
-      _counter++;
-    });
-  }
+  String searchQuery = '';
+  
+  get child => null; //user input
```
Comment (erickjtorres) [inline lib/main.dart]: I don't think you need this what is this used for?

**Review by erickjtorres — CHANGES_REQUESTED** [review]
Looks good so far nice work! Just had a couple of nit comments. 

@bridgettorres @inikasingh21 please remove any unnecessary comments or try to avoid any commented code in the future. It makes it harder to read the code

**Review by erickjtorres — APPROVED** [review]

File: pubspec.yaml [hunk 5]
```diff
@@ -59,7 +60,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
+  assets:
+      - assets/logo.png
   #   - images/a_dot_burr.jpeg
```
Comment (bridgettorres) [inline pubspec.yaml:65]: Remove unneeded dependencies

**Review by bridgettorres — COMMENTED** [review]
LGTM

**Review by bridgettorres — APPROVED** [review]

**Pull request merged**